docs/docs/creating-a-transformer-plugin.md

```diff
@@ -1,44 +1,41 @@
+- **Source** _plugins_, ou _plugins_ de fonte de dados, proveem informação de uma fonte remota ou local para o que o Gatsby chama de [nós](/docs/node-interface/).
+- **Transformer** _plugins_, ou _plugins_ de transformação, "transformam" informação provida por _source plugins_ para novos nós e/ou para campos dos nós.
 
-The purpose of this doc is to:
+O objetivo deste documento é:
 
-1.  Define what a Gatsby transformer plugin is, and
-2.  Walk through a simplified reimplementation of an existing plugin, to demonstrate how to create a transformer plugin.
+1. Definir o que é um _plugin_ de transformação do Gatbsy, e
+2. Acompanhar uma reimplementação simplificada de um _plugin_ existente para demonstrar como criar um _plugin_ de transformação.
 
-## What do transformer plugins do?
+## O que _plugins_ de transformação fazem?
 
-Transformer plugins "transform" data of one type into another type. You'll often use both source plugins and transformer plugins in your Gatsby sites.
+_Plugins_ de transformação "transformam" dados de um tipo para outro tipo. Você frequentemente usará ambos os plugins de fonte de dados e de transformação no seu website feito com Gatsby.
 
-This loose coupling between the data source and transformer plugins allow Gatsby developers to quickly assemble complex data transformation pipelines with little work.
+Esse acoplamento flexível entre os _plugins_ de fonte de dados e de transformação permitem que os desenvolvedores que usam Gatsby montem rapidamente pipelines de transformação de dados complexos com pouco trabalho.
 
-## How do you create a transformer plugin?
+## Como criar um _plugin_ de transformação?
 
-Just like a source plugin, a transformer plugin is a normal NPM package. It has a `package.json` file with optional dependencies as well as a `gatsby-node.js` file where you implement Gatsby's Node.js APIs.
+Assim como o _plugin_ de fonte dados, o _plugin_ de transformação é um pacote NPM normal. Ele possui um arquivo `package.json` com dependências opcionais assim como um arquivo `gatsby-node.js` onde você implementa as APIs Node.js do Gatsby.
 
-`gatsby-transformer-yaml` is transformer plugin that looks for new nodes with a media type of text/yaml (e.g. a .yaml file) and creates new YAML child node(s) by parsing the YAML source into JavaScript objects.
+`gatsby-transformer-yaml` é um _plugin_ de transformação que procura por novos nós com o _media type_ igual a _text/yaml_ (e.g. um arquivo .yaml) e cria novo(s) nó(s) filho(s) do tipo YAML através do processamento  do arquivo fonte YAML em objetos JavaScript.
 
-Check out this example of rebuilding a simplified `gatsby-transformer-yaml` directly in a site. Say you have a default Gatsby starter site which includes a `src/data/example.yml` file:
+Confira este exemplo de reconstrução simplificada de um `gatsby-transformer-yaml` diretamente em um site. Digamos que você tenha um site inicial padrão do Gatsby que inclua um arquivo `src/data/example.yml`:
```

Comment:
faltou itálico em site

Reply:
tem certeza que starter está sendo traduzido como "site inicial padrão"?  @jessescn , pode nos ajudar?

Reply:
No [Google sheets](https://docs.google.com/spreadsheets/d/1TLztI-jrPslCt0fgk4B9tGnhhS_7JQZJfMql1ePsIZo/edit#gid=0) existe a seguinte tradução sugerida: `starter / site a partir de um starter`. Posso mudar para:
```
Digamos que você tenha um website a partir do Gatsby _starter_ que inclua um arquivo `src/data/example.yml`:
```
O que acha?